maquetado/scss/titles.scss

```diff
@@ -0,0 +1,13 @@
+@import 'styles';
+
+.title {
+  background-image: url(assets/triangle.svg);
```

Comment:
you must use $title-decoration-height and $title-decoration-width for the image dimensions.
Try using the :before pseudo element